### PR TITLE
Remove bastion requirement from katello

### PR DESCRIPTION
**Pull request by jlsherrill**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by jlsherrill** [comment]
Note: this requires https://github.com/Katello/katello/pull/8032 

which is not merged yet.

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
doh 

[test rpm]

**Comment by jlsherrill** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
@ehelms seeing this on the build now: 
Gem loading error: Could not find 'bastion' (< 7.0.0, >= 6.1.22) among 153 total gem(s)
I can't quite tell if its pulling the newest gem or not, since the latest build ci is failing (i assume that's where it gets it from?)  https://ci.theforeman.org/job/katello-master-release/

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -29,9 +29,8 @@ Requires: %{?scl_prefix_ror}rubygem(rails)
 Requires: %{?scl_prefix_ruby}rubygem(json)
 Requires: %{?scl_prefix}rubygem(activerecord-import)
 Requires: %{?scl_prefix}rubygem(anemone)
+Requires: %{?scl_prefix}rubygem(angular-rails-templates)
```
Comment (ehelms) [inline packages/katello/rubygem-katello/rubygem-katello.spec]: I think this needs to be a build requires too

**Comment by ehelms** [comment]
This is a bit of a chicken'n'egg problem. I'll test it locally.

**Comment by jlsherrill** [comment]
@ehelms updated build requires, yeah i suspected it might only be pulling the latest gem from the last successful build, but couldn't find where that was happening exactly

**Pull request merged**